nss-c31-mixed-havoc/nutshell
# Bm save friend

# Description

I have added a button at the bottom of the friends section to add a new friend. When clicked, it will open an input box. The user will enter someones name to add them as a friend. The code makes sure that a user by that name exists and that they are not already friends before adding them.

Applies to issue #8 

## Type of change

Please delete options that are not relevant.


# Testing Instructions

1. `git fetch --all`
2. `git checkout bm-save-friend`
3. cd into /src/lib and run `grunt`
4. load `http://localhost:8080/` , log into the page with anyone's info, and verify that there is now a save button at the bottom of the page and that you can save a user who you are not friends with. This is not case sensitive. Also, for fun, try adding yourself as a friend.



# Checklist:
